backend/census_historical_migration/sac_general_lib/audit_information.py

```diff
@@ -0,0 +1,150 @@
+from census_historical_migration.workbooklib.census_models.census import (
+    CensusGen22 as Gen,
+    CensusCfda22 as Cfda,
+    CensusFindings22 as Finding,
+)
+from census_historical_migration.base_field_maps import FormFieldMap, FormFieldInDissem
+from census_historical_migration.sac_general_lib.utils import (
+    _create_json_from_db_object,
+)
+import audit.validators
+
+DOLLAR_THRESHOLD = 750000

```

Comment:
It would be completely reasonable to move this out to the app-level `config`, for a variety of reasons. Perhaps not part of this commit, but we are going to need this number elsewhere.